src/web/src/pages/signup.tsx

```diff
@@ -153,6 +153,14 @@ const useStyles = makeStyles((theme: Theme) =>
   })
 );
 
+enum SIGN_UP_STEPS {
+  OVERVIEW = 0,
+  BASIC_INFO = 1,
+  GITHUB_ACCOUNT = 2,
+  RSS_FEEDS = 3,
+  REVIEW = 4,
+}
```

Comment:
You could drop the initializers, as first value starts from 0 and next values are automatically incremented.
```suggestion
  OVERVIEW,
  BASIC_INFO,
  GITHUB_ACCOUNT,
}
```

Reply:
Sure.